pkg/label/k8s.go

```diff
@@ -1,6 +1,8 @@
 package label
 
 const (
+	// MasterNode label denotes a K8s cluster master node.
+	MasterNode = "node.kubernetes.io/master"
 	// MasterNodeRole label denotes K8s cluster master node role.
 	MasterNodeRole = "node-role.kubernetes.io/master"
```

Comment:
Any reason why you did not remove the MasterNodeRole here?

Reply:
I thought it was still in use somewhere but I guess it is not.